docs/Best practices/open-source-project-launch-checklist.md

```diff
@@ -0,0 +1,71 @@
+# Open Source Project Launch Checklist
+
+Gefeliciteerd! Je leest dit omdat je van plan bent om je project te open sourcen!
```

Comment:
Ik vind dit ook wel mooi, hoor, elke zin op een eigen regel. In de Contributing Guide staat dat we voor een line wrap gaan van 120 chars per alinea. Wat doen we? Volgen we twee standaarden? Ik weet niet zo goed wat hierin het beste is ...

Reply:
Line-break op 120 chars is voor mij meer gedoe, ik heb mijn editor standaard ingesteld op 80. Dus als dit acceptabel is scheelt het mij werk. Maar eens dat consistentie helpt.

Reply:
Ik gebruik VSCode en daarvoor heb ik een [workspace specifieke config](https://github.com/ospo-nl/kennisbank/blob/main/.vscode/settings.json) ingecheckt 😉

Reply:
Voor nu schiet ik het er gewoon in, volgende bijeenkomst maar even over hebben.